cos-nvidia-gpu-installer/installer.sh

```diff
@@ -216,4 +226,27 @@ main() {
     post_installation_sequence
 }
 
+usage() {
+  echo "${0} [-d]"
+  echo "  -d:  Device plugin feature is enabled on the node."
```

Comment:
Instead of adding an option like this, let's just use environment variables.

The script is already configurable via environment variables: 
https://github.com/ContainerEngine/accelerators/blob/v0.4/cos-nvidia-gpu-installer/installer.sh#L28

You can add `DEVICE_PLUGIN_ENABLED` there as well. And then pass that environment variable from the daemonset or whereever you call this from.

Reply:
Done. PTAL.